### PR TITLE
Bump version to 1.27.0

### PR DESCRIPTION
**Pull request by bquorning**
Before submitting the PR make sure the following are checked:

* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [ ] Added tests.
* [x] Added an entry to the [changelog](https://github.com/rubocop-hq/rubocop-rspec/blob/master/CHANGELOG.md) if the new code introduces user-observable changes.
* [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

**Comment by bquorning** [comment]
I was probably too early here. We should merge #625 first, then rebase this branch.

**Comment by bquorning** [comment]
…and @dgollahon mentioned a potential issue with `RSpec/EmptyLineAfterHook`: https://github.com/rubocop-hq/rubocop-rspec/pull/632#issuecomment-396821220

**Comment by bquorning** [comment]
#625 has been merged, and this branch is rebased. I think we wait with fixing the potential `RSpec/EmptyLineAfterHook`, and make this release now.

**Comment by bquorning** [comment]
Absolutely, I can wait. And good call on not releasing anything with known bugs. 👍

**Comment by bquorning** [comment]
> @bquorning If you're willing to wait til tonight I can try to get a quick fix in for #632 (comment). I like avoiding issues we're aware of before releasing if we can.

No problem, there’s no urgency.

**Comment by bquorning** [comment]
@dgollahon Time for a release?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Master (Unreleased)
 
+## 1.27.0 (2018-06-13)
```
Comment (Darhazer) [inline CHANGELOG.md]: 2018-06-14 ;)

Comment (bquorning) [reply CHANGELOG.md]: Right you are.

**Comment by dgollahon** [comment]
🎉

**Pull request merged**